ansible_collections/arista/avd/examples/l2ls-fabric/README.md

```diff
@@ -0,0 +1,418 @@
+
+    ``` yaml
+    ---
+    ### group_vars/DC1_SPINES.yml
+
+    type: l3spine
+    ```
+
+=== "DC1_FABRIC.yml"
+
+    Update with the following changes and additions.
+
+    - Change the node key **spine** to **l3spine** to match the node type set previously in DC1_SPINES.yml
+    - Add **loopback_ipv4_pool**
+    - Add **mlag_peer_l3_ipv4_pool**
+    - Add **virtual_router_mac_address**
+
+    Update DC1_FABRIC.yml with the following recommended settings. Use your own IP pools.
+
+    ``` yaml
+    # Node Key must be l3spine to match type
+    l3spine:
+      defaults:
+        platform: cEOS-LAB
+        spanning_tree_mode: mstp
+        spanning_tree_priority: 4096
+        # Loopback is used to generate a router-id
+        loopback_ipv4_pool: 1.1.1.0/24
+        mlag_peer_ipv4_pool: 192.168.0.0/24
+        # Needed for L3 peering across the MLAG Trunk
+        mlag_peer_l3_ipv4_pool: 10.1.1.0/24
+        # Used for SVI Virual MAC address
+        virtual_router_mac_address: 00:1c:73:00:dc:01
+        mlag_interfaces: [Ethernet47, Ethernet48]
+    ```
+
+=== "DC1_NETWORK_SERVICES.yml"
+
+    Update Network Services to use L3 SVIs.
+
+    ???+ Note
+
+        To create L3 SVIs on the spines, we need to utilize an L3 VRF. In our case, we will use the default VRF. `MY_FABRIC` is simply a tenant name for organizing VRFs and SVIs.
+
+    ``` yaml
+    ---
+    tenants:
+      MY_FABRIC:
+        vrfs:
+          default:
+            svis:
+              10:
+                name: 'BLUE-NET'
+                tags: [bluezone]
+                enabled: true
+                ip_virtual_router_addresses:
+                  - 10.10.10.1
+                nodes:
+                  SPINE1:
+                    ip_address: 10.10.10.2/24
+                  SPINE2:
+                    ip_address: 10.10.10.3/24
+              20:
+                name: 'GREEN-NET'
+                tags: [greenzone]
+                enabled: true
+                ip_virtual_router_addresses:
+                  - 10.20.20.1
+                nodes:
+                  SPINE1:
+                    ip_address: 10.20.20.2/24
+                  SPINE2:
+                    ip_address: 10.20.20.3/24
+              30:
+                name: 'ORANGE-NET'
+                tags: [orangezone]
+                enabled: true
+                ip_virtual_router_addresses:
+                  - 10.30.30.1
+                nodes:
+                  SPINE1:
+                    ip_address: 10.30.30.2/24
+                  SPINE2:
+                    ip_address: 10.30.30.3/24
+    ```
+
+Now re-run your playbook and build the new configs. The intended/configs for the spines will have been updated with L3 SVIs.
+
+```bash
+ansible-playbook playbooks/build.yml
+```
+
+If you wish to deploy these changes, then simply run the deploy playbook.
+
+```bash
+ansible-playbook playbooks/deploy.yml
+```
+
+## Next steps
+
+Try building your own topology.
```

Comment:
Add single trailing newline to end of file

Reply:
newline already exists.  Line 419